src/layers/proprietary.md

```diff
@@ -15,7 +15,7 @@ Let's add a Bing layer.
   ```js
     source: new ol.source.BingMaps({
       imagerySet: 'Road',
-      key: 'Ak-dzM4wZjSqTlzveKz5u0d4IQ4bRzVI309GxmkgSVr1ewS6iPSrOvOKhA-CJlm3'
+      key: 'AkGbxXx6tDWf1swIhPJyoAVp06H0s0gDTYslNWWHZ6RoPqMpB9ld5FY1WutX8UoF'
```

Comment:
Didn't we want to remove API keys from the workshops? I may be wrong, but I could swear that a pull request from @tschaub got merged a while ago, which removed the API keys.